crates/solidity/testing/snapshots/cst_output/Expression/overlapping_operators/generated/0.4.11.yml

```diff
@@ -0,0 +1,19 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+Source: >
+  1  │ a & b && c                                                                       │ 0..10
```

Comment:
I suggest also adding a test for the opposite: `a && b & c`

To make sure the parser doesn't consume the first `&` of `&&` as a bitwise operator, and error out on the second.

Reply:
In terms of precedence, that's a valid test, but in this PR, order isn't the issue. I'll add more precedence test soon in a new PR